testgrid/cmd/transfigure/README.md

```diff
@@ -9,10 +9,10 @@ To have your instance of Prow push to TestGrid automatically, add a [Prow Job](/
 similar to the one below:
 ```yaml
 postsubmits:
-  YOUR-ORG/YOUR-REPO: # Replace this
-  - name: post-test-infra-upload-testgrid-config
-    cluster: test-infra-trusted # You may need to specify a cluster, depending on your Prow config
-    run_if_changed: '^(dir/to/your/prowjobs/.*\.yaml)|(dir/to/your/testgrid/default\.yaml)$' # Replace this
+  YOUR-ORG/YOUR-REPO: # TODO: Replace this
+  - name: post-[repo-name]-upload-testgrid-config # TODO: Rename this 
+    cluster: test-infra-trusted # TODO: You may need to specify a cluster, depending on your Prow config
```

Comment:
Should we not default to test-infra-trusted?

Reply:
Do people create the job on k8s prow or other prow?

Reply:
Other Prows. They may not have or use a trusted cluster, or may have a different alias for their cluster.